tests/test_volume_loading.py

```
@@ -95,12 +95,126 @@ def test_non_bind_mounts_should_fail():
+
+    try:
+        runner = _volume_dot_path_runner('tests/data/usage_scenarios/volume_load_non_bind_mounts.yml', allowed_volume_mounts=['gmt-test-volume-delete-me'])
+
+        with Tests.RunUntilManager(runner) as context:
+            context.run_until('setup_services')
+    finally:
+        subprocess.check_output(['docker', 'volume', 'rm', 'gmt-test-volume-delete-me'])
+
+def test_allowed_volume_fails_bc_readonly():
+
+    subprocess.check_output(['docker', 'volume', 'create', 'gmt-test-volume-delete-me'])
```

Ruff (0.15.9)

[error] Starting a process with a partial executable path

(S607)